doc/api/process.md

```diff
@@ -637,6 +637,28 @@ process's [`ChildProcess.disconnect()`][].
 If the Node.js process was not spawned with an IPC channel,
 `process.disconnect()` will be `undefined`.
 
+## process.dlopen(module, filename[, flags])
+<!-- YAML
+added: REPLACEME
```

Comment:
@addaleax `process.dlopen` was added in v2.0.0. It was called `node.dlopen` before I think. What should we claim?